packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -2982,6 +2982,204 @@ Generated by [AVA](https://ava.li).
     ␊
     (Netlify Build completed in 1ms)`
 
+## Erroneous headers created by a build command are handled
```

Comment:
Wondering if this snapshot should generate a similar warning than the one below? - https://github.com/netlify/build/pull/3422/files#diff-b8b637f5f15aedc2854a2ef8877b0da0fd3b5c01de20fcc3780a8e8102e7a02bR3177

Reply:
Good catch. :+1: 

We only print a warning message when `headers`/`redirects` with invalid syntax are added either:
  - From the initial `_headers`/`_redirects`/`netlify.toml`
  - Through `netlifyConfig.headers|redirects`

We do not do so when `_headers`/`_redirects` has been added by the build command. This is a missing feature, which I will fix in a follow up PR.